showcase/src/assets/translations/en.json

```diff
@@ -98,10 +98,15 @@
           "INPUT_ONLY_NUMBERS": "Accept only numbers",
           "INPUT_ALPHANUMERICAL_CHARACTERS": "Accept only alphanumeric characters",
           "INPUT_NO_SPECIAL_CHARACTERS": "Accept all except special characters",
-          "INPUT_UPPERCASE_CHARACTERS": "Accept only uppercase characters",
-          "TYPE_A_VALUE": "Type a value"
+          "INPUT_UPPERCASE_CHARACTERS": "Accept only uppercase characters"
         },
-        "TITLE": "Keyboard directive"
+        "TRANSFORM_INPUT": {
+          "TITLE": "Transform Input Directive",
+          "ONLY_UPPER_CASE_LABEL": "Upper case only",
+          "ONLY_LOWER_CASE_LABEL": "Lower case only"
```

Comment:
Same here

Also you can add in this to the label "(ngModel)" to make it clear that a simple input with ngModel is used for that field